doc/source/reference/routines.indexing.rst

```diff
@@ -41,6 +41,7 @@ Indexing-like operations
    diag
    diagonal
    select
+   as_strided
```

Comment:
Ah, that's a bummer. I tried configuring a doc build on ReadTheDocs (see readthedocs.org/projects/numpy) but have not been able to get numpy building yet. I'm glad you figured out another way :)